qa/rpc-tests/wallet.py

```diff
@@ -6,6 +6,27 @@
 from test_framework.test_framework import BitcoinTestFramework
 from test_framework.util import *
 
+def check_array_result(object_array, to_match, expected):
```

Comment:
If you copied this function from elsewhere, wouldn't it make sense to *move* it instead to `test_framework/util.py`, so the utility function can be used in other tests as well?